### ISSUE TITLE
Flag up internally when a legal block shows up only on a filtered line

### ISSUE THREAD

**Issue by alexhaydock** [openrightsgroup/blocked-org-uk#310]
Looking at the legal blocks page, we aren't flagging up when a site is being blocked on an ISP's filtered line, but **isn't** being blocked on their unfiltered one. Situations where this is happening could potentially indicate that this is an issue with an ISP filter and not a legal block.

For instance, this example from the legal blocks page:
![image](https://i.imgur.com/EimuUCq.png)

The bottom site is blocked on both Plusnet Unfiltered and Plusnet Filtered (good sign of a legal block), but the top site seems to be blocked on Plusnet Filtered but not the Unfiltered line.

Flagging these up somewhere might be useful for seeing which blocks need more investigation, or for highlighting errors in how the ISPs are implementing the blocks.

**Comment by edjw** [comment]
@dantheta – I suppose we could make a list in the backend of sites under legal blocks that are available on an ISP's filtered connection but not on the unfiltered one?

**Comment by dantheta** [comment]
Yep - I think this one would suit a periodic report (output to a file or a saved list on the site) rather than a realtime notification.

**Comment by edjw** [comment]
I think a saved list on the site would be best.
